setup.py

```diff
@@ -13,9 +13,6 @@
     print("To install, run 'python setup.py install'\n")
 
 if sys.argv[-1] == 'publish':
-    if sys.version_info.major < 3:
```

Comment:
I'm confused on this we need to remove this right ? because we are dropping support of python2